src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +108,11 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        CandidateValidator candidateValidator = new CandidateValidator();
+        return candidates.stream()
+                .filter(candidateValidator)
```

Comment:
```suggestion
        return candidates.stream()
                .filter(new CandidateValidator())
```
java is a verbose language in its own right, we should simplify it as much as possible :)